docs/addons/cert-manager.md

```diff
@@ -0,0 +1,79 @@
+# cert-manager Add-on
+
+This add-on installs [cert-manager](https://github.com/cert-manager/cert-manager).
+
+cert-manager adds certificates and certificate issuers as resource types in Kubernetes clusters, and simplifies the process of obtaining, renewing and using those certificates.
+It can issue certificates from a variety of supported sources, including Let's Encrypt, HashiCorp Vault, and Venafi as well as private PKI.
+It will ensure certificates are valid and up to date, and attempt to renew certificates at a configured time before expiry.
+
+## Usage
+
+```typescript
+
+import 'source-map-support/register';
+import * as cdk from 'aws-cdk-lib';
+import * as blueprints from '@aws-quickstart/eks-blueprints';
+
+const app = new cdk.App();
+
+const addOn = new blueprints.addons.CertManagerAddOn()
+
+const blueprint = blueprints.EksBlueprint.builder()
+  .addOns(addOn)
+  .build(app, 'my-stack-name');
+```
+
+## Configuration Options
+
+- `version`: Version fo the Helm Chart to be used to install cert-manager

```

Comment:
let's drop version from here - it is common for all helm add-ons.

Reply:
done